### DISCUSSION TITLE
Separate scheduled vs. ad-hoc silences in Grafana 8

### DISCUSSION THREAD

**Discussion by mmabo80** [grafana/grafana#49303] [Dashboards, Panels & Visualizations]
Our end users are concerned with the scheduled and ad-hoc silences being available for configuration in the same view. Specifically, they would prefer scheduled silences to be displayed on a separate pane with separate access controls so that users do not disable or change pre-scheduled silences in error when creating ad-hoc silences.

Please confirm whether this feature request can be accommodated and when it would be available. Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
